src/app/pages/list/multiselect-list/multiselect-list.component.scss

```diff
@@ -47,3 +47,8 @@ footer.active {
         min-width: 0;
     }
 }
+
+::ng-deep .mat-checkbox-inner-container {
```

Comment:
I'm a little hesitant to put !important here.  Can you try using slightly more specific css?  Something like this perhaps:

```<mat-list class="multi-select-list">```

```
 ::ng-deep .multi-select-list .mat-checkbox-inner-container {
    width: 18px;
    height: 18px;
}
```